src/main/java/gov/cabinetoffice/gap/applybackend/web/GrantSchemeController.java

```diff
@@ -41,7 +42,9 @@ public ResponseEntity<GetGrantSchemeWithApplicationAndAdverts> getGrantSchemeByI
         final GetGrantSchemeDto grantSchemeDto = new GetGrantSchemeDto(grantScheme);
         final GetGrantApplicationDto grantApplicationDto = grantSchemeMapper.grantSchemeToGetGrantApplicationDto(grantScheme);
         final List<GetGrantAdvertDto> grantAdvertDtos = grantScheme.getGrantAdverts().stream()
-                .map(grantAdvert -> grantAdvertService.grantAdvertToDto(grantAdvert, jwtPayload.getSub(), grantSchemeId))
+                .map(grantAdvert ->
+                        grantAdvert.getStatus().equals(GrantAdvertStatus.PUBLISHED) ?
+                        grantAdvertService.grantAdvertToDto(grantAdvert, jwtPayload.getSub(), grantSchemeId) : null)
```

Comment:
won't this end up putting null values in the list? Not sure if that matters later on or not but if it does its probably not ideal